packages/window-size-tool/rules.json

```
@@ -33,8 +33,7 @@
   {
     "width": "> 576 and <= 768",
     "terms": [
-      "count(//CLUSTER[@NUM_COLS > 1 ]) = 0 and count(//CLUSTER) > 0",
-      "count(//LIST[count(FIELD | CONTAINER) <= 2]) > 0 and count(//LIST[count(FIELD | CONTAINER) > 2]) = 0"
+      "count(//CLUSTER[@NUM_COLS > 1 ]) = 0 and count(//LIST[count(FIELD | CONTAINER) > 2]) = 0"
```

This is now modifying files that don't contain any UIM markup, like those that import VIM or JSP scripts. Might be best to continue to check for either of those elements

Something like this?
```suggestion
      "count(//CLUSTER[@NUM_COLS > 1 ]) = 0 and count(//LIST[count(FIELD | CONTAINER) > 2]) = 0 and (count(//CLUSTER) > 0 or count(//LIST) > 0)"
```

---

Yes, that should work, I am just running the tool again on CEF to test it.